apps/docs/app/api/revalidate/route.ts

```diff
@@ -35,14 +35,12 @@ export async function _handleRevalidateRequest(request: NextRequest) {
   }
 
   let authorizationLevel = AuthorizationLevel.Unauthorized
-
-  const token = authorization.replace(/^Bearer\s+/, '')
+  const token = authorization.replace(/^Bearer /, '')
```

Comment:
double-checked the RFC and seems to only allow 1 * SP here so tightening this validation